From 9823fb93f79b6c38b28cfec89f427e2a0472436a Mon Sep 17 00:00:00 2001
From: Greg Kroah-Hartman <gregkh@google.com>
Date: Wed, 15 Jul 2020 09:40:13 +0200
Subject: [PATCH 770/830] ANDROID: GKI: Enable CONFIG_BT_HIDP as it is needed
 by some platforms
X-NVConfidentiality: public

Might as well build it in so we don't have to worry about ABI issues
over time.

Bug: 157965270
Bug: 162909109
Signed-off-by: Greg Kroah-Hartman <gregkh@google.com>
Change-Id: I285bf3e3bf4bb9c649c3ac667834c6d12c7e1f4a
Signed-off-by: Connor O'Brien <connoro@google.com>
---
 arch/arm64/configs/gki_defconfig | 1 +
 arch/x86/configs/gki_defconfig   | 1 +
 2 files changed, 2 insertions(+)

diff --git a/arch/arm64/configs/gki_defconfig b/arch/arm64/configs/gki_defconfig
index 917b4cc4db11..d38bc7355e53 100644
--- a/arch/arm64/configs/gki_defconfig
+++ b/arch/arm64/configs/gki_defconfig
@@ -207,6 +207,7 @@ CONFIG_NET_CLS_ACT=y
 CONFIG_VSOCKETS=y
 CONFIG_BPF_JIT=y
 CONFIG_BT=y
+CONFIG_BT_HIDP=y
 CONFIG_CFG80211=y
 # CONFIG_CFG80211_DEFAULT_PS is not set
 # CONFIG_CFG80211_CRDA_SUPPORT is not set
diff --git a/arch/x86/configs/gki_defconfig b/arch/x86/configs/gki_defconfig
index bd21f40fd889..1c75172e66c6 100644
--- a/arch/x86/configs/gki_defconfig
+++ b/arch/x86/configs/gki_defconfig
@@ -185,6 +185,7 @@ CONFIG_NET_CLS_ACT=y
 CONFIG_VSOCKETS=y
 CONFIG_BPF_JIT=y
 CONFIG_BT=y
+CONFIG_BT_HIDP=y
 CONFIG_CFG80211=y
 # CONFIG_CFG80211_DEFAULT_PS is not set
 # CONFIG_CFG80211_CRDA_SUPPORT is not set
-- 
2.17.1

